.github/workflows/build.yml

```
@@ -8,7 +8,10 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        java_version: [11, 17, 18, 19-ea]
+        java_version: [11, 17, 18]
+        include:
+          - java_version: 19-ea
+            experimental: true
```

```suggestion
            experimental: true
    continue-on-error: ${{ matrix.experimental }}
```
I think you need that line to allow the overall build to succeed even if `19-ea` fails.  See, e.g., here: https://github.com/google/error-prone/blob/f94a04d407583bc9f85847089d72298deecf6fb0/.github/workflows/ci.yml#L34